docs/shared/next-plugin.md

```diff
@@ -60,7 +60,7 @@ myorg/
 
 ## See Also
 
-- [Using Next.js](/{{framework}}/plugins_next/overview)
```

Comment:
This line should stay the same. This should point to a guide that is specific for using Next.js with Nx.

Reply:
Then there is a problem, it redirects to the same page, so even if you click nothing happens.
Also, the heading says See Also, which means we are saying something apart from this page, either we should remove this entire See Also section or redirect somewhere else.

Reply:
I see. Thanks :)